src/cli/color.js

```diff
@@ -1,6 +1 @@
-import _ from 'lodash';
-import ansicolors from 'ansicolors';
-
-export const green = _.flow(ansicolors.black, ansicolors.bgGreen);
-export const red = _.flow(ansicolors.white, ansicolors.bgRed);
-export const yellow = _.flow(ansicolors.black, ansicolors.bgYellow);
+export { green, red, yellow } from 'chalk';
```

Comment:
Why get rid of the backgrounds? 😢 Chalk supports them just fine

Reply:
True. I can put them back in.